### PR TITLE
fix(learn): Add for and id attributes to label and input

### PR DESCRIPTION
**Pull request by ashleeradka**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

The lesson prior to this mentions the best practice of explicitly defining the relationship between the label and its input, I've added a `for` attribute to the `<label>` elements and an `id` attribute to the corresponding `<input>` elements to align with best practices. 

- [x] Tested changes locally

### CODE REVIEW

**Review by MrSlimCoder — DISMISSED** [review]

**Review by MrSlimCoder — DISMISSED** [review]

**Comment by ahmadabdolsaheb** [comment]
@ashleecrusco  thanks for the pr.
@RandellDawson would these changes improve the challenge?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.english.md [hunk 1]
```diff
@@ -76,11 +76,11 @@ tests:
...
-    <label><input type="radio" name="indoor-outdoor"> Indoor</label>
-    <label><input type="radio" name="indoor-outdoor"> Outdoor</label><br>
-    <label><input type="checkbox" name="personality"> Loving</label>
-    <label><input type="checkbox" name="personality"> Lazy</label>
-    <label><input type="checkbox" name="personality"> Energetic</label><br>
+    <label for="indoor"><input id="indoor" type="radio" name="indoor-outdoor"> Indoor</label>
+    <label for="outdoor"><input id="outdoor" type="radio" name="indoor-outdoor"> Outdoor</label><br>
+    <label for="loving"><input id="loving" type="checkbox" name="personality"> Loving</label>
+    <label for="lazy"><input id="lazy" type="checkbox" name="personality"> Lazy</label>
+    <label for="energetic"><input id="energetic" type="checkbox" name="personality"> Energetic</label><br>
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.english.md:83]: @ashleecrusco You have added the checkboxes to the challenge seed code.  The user is supposed to create these.  In the solution, it is fine, but these need to be deleted.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ashleecrusco See my suggested change.  Also, you have a merge conflict that needs to be resolved.  If you have any questions, please let us know.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.english.md (hunk 1 above)
Comment (ashleecrusco) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.english.md:83]: @RandellDawson I'm not sure I understand 🤔 I've only added `for` and `id` here. The type of `checkbox` was already provided. This exercise asks for the value attribute to be added, is that right?

**Comment by moT01** [comment]
I fixed up some merge conflicts, and a typo here. I think this should be added since the challenges after it include the `for` and `id` on the inputs.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
With @moT01's updates, I am fine with this PR now.

@Sky020 Could you review this one.  Feel free to merge if you are ok with changes.  Thanks.

**Review by ShaunSHamilton — APPROVED** [review]
Hi @ashleecrusco . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**